src/common/config_parser.cpp

```diff
@@ -270,6 +270,25 @@ void ConfigParser::addOptionsModel(po::options_description& desc) {
      "Number of decoder layers (s2s)")
     //("dec-high-context", po::value<std::string>()->default_value("none"),
     // "Repeat attended context: none, repeat, conditional, conditional-repeat (s2s)")
+
+    // Multi-head attention
```

Comment:
I think these only apply to s2s models, and not transformer, which already has multi-head attention.

Can you please make that clear in this comment?

Can you please also review the names of your options to make sure they are consistent with the names used fro transformer?